functions-and-scope.js

```diff
@@ -67,11 +127,26 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // Schrijf een script die op basis van de grades array (hierboven) checkt wat het hoogst behaalde cijfer is. Je mag hier geen bestaande methoden voor gebruiken. Schrijf de stapjes eerst uit en vraag jezelf de volgende dingen af:
 // * Hoe kan ik iedere waarde van de array langsgaan?
 // * Op welke conditie moet ik checken?
-// * Hoe zorgt ik ervoor dat wanneer ik een cijfer tegenkom die aan de conditie voldoet, ik dit ergens kan opslaan?
+// * Hoe zorg ik ervoor dat wanneer ik een cijfer tegenkom die aan de conditie voldoet, ik dit ergens kan opslaan?
 // Log het antwoord in de terminal.
 
 // ---- Verwachte uitkomst: 9
 
+console.log("\nOpdracht 3a:")
+//Stappen
+// * Hoe kan ik iedere waarde van de array langsgaan? met een for loop
+// * Op welke conditie moet ik checken? of het eerste cijfer hoger is dan het 2e en dan het 2e hoger is dan het 3e,....
+// * Hoe zorg ik ervoor dat wanneer ik een cijfer tegenkom die aan de conditie voldoet, ik dit ergens kan opslaan? door een var te benoemen > dit werkte niet helemaal, want ik kreeg eerst 8 terug. Ik moest de oude waarde van highest ook nog vergelijken met de nieuwe waarde van highest.
+
+let highest = 0;
+for (let i = 0; i < grades.length - 1; i++) {
+    if ((grades[i] > grades[i + 1]) && (highest < grades[i])) {
+        highest = grades[i];
+    } else if ((grades[i + 1] > grades[i]) && (highest < grades[i + 1])) {
+        highest = grades[i + 1];
+    }
```

Comment:
Dit ziet er wel erg complex uit, als het goed is moet dit ook werken
```suggestion
    if (grades[i] > highest) {
        highest = grades[i];
    }
```